libsolidity/ast/ASTForward.cpp

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
leftover?

Reply:
it's from the "fix macos" commit which is temporary. will be properly integrated now